cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -319,12 +319,13 @@ features:
         inputPlaceholder: Destination Column
         inputSuffix: _copy
         label: Copy to a new column
-      dataErrorMessageTitle: Sorry, we could not load data for "{workspaceName}"
+      dataErrorMessageTitle: Sorry, the data cannot be loaded.
```

Comment:
May be something simpler like`Unable to load data. Please refresh to try again`? @bdmogal we already use 

```
Unable to copy data.. 
Unable to create copy task
...
```

Reply:
Will consult with @leacd on this as well